### PR TITLE
Update UWP Project Template

### LINKED ISSUE #4983: Can't create new Windows 10 UWP app
Similar issue to: https://github.com/mono/MonoGame/issues/4435 however uninstalling & reinstalling the Universal Windows App Development Tools did NOT solved the issue.  I am able to create & publish normal windows 10 apps without any issue.

Running the latest version of MonoGame 3.5.1

_Error: the vstemplate file references the wizard class 'Microsoft.VisualStudio.WinRT.TemplateWizards.ApplicationInsights.Wizard' which does not exsist in the assembly 'Microsoft.VisualStudio.WinRT.TemplateWizards, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'._

### PR DESCRIPTION
**Pull request by cra0zy**
Fixes #4983

### CODE REVIEW

File: ProjectTemplates/VisualStudio2015/WindowsUniversal/WindowsUniversal.vstemplate [hunk 1]
```diff
@@ -48,10 +48,6 @@
     <FullClassName>Microsoft.VisualStudio.WinRT.TemplateWizards.CreateProjectCertificate.Wizard</FullClassName>
   </WizardExtension>
   <WizardExtension>
-    <Assembly>Microsoft.VisualStudio.WinRT.TemplateWizards, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</Assembly>
-    <FullClassName>Microsoft.VisualStudio.WinRT.TemplateWizards.ApplicationInsights.Wizard</FullClassName>
-  </WizardExtension>
-  <WizardExtension>
```
Comment (tomspilman) [inline ProjectTemplates/VisualStudio2015/WindowsUniversal/WindowsUniversal.vstemplate]: How did you determine this should be removed?

I just checked my system and this is still there, but the class name is:

`<FullClassName>Microsoft.VisualStudio.WinRT.TemplateWizards.CreateProjectCertificate.Wizard</FullClassName>`

Comment (cra0zy) [inline ProjectTemplates/VisualStudio2015/WindowsUniversal/WindowsUniversal.vstemplate]: You need to update your SDK, the newest templates don't have it and with it in you can't create the project template at all.

Comment (tomspilman) [inline ProjectTemplates/VisualStudio2015/WindowsUniversal/WindowsUniversal.vstemplate]: I did.  I have VS Update 3 installed.

I was looking at `C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\IDE\ProjectTemplates\CSharp\Windows Root\Windows UAP\1033\BlankApplication\BlankApplication.vstemplate`.

Here is the content of it:  https://gist.github.com/tomspilman/c266c20490c88ef709e24f6c79641322

Comment (cra0zy) [inline ProjectTemplates/VisualStudio2015/WindowsUniversal/WindowsUniversal.vstemplate]: Your template doesn't have `Microsoft.VisualStudio.WinRT.TemplateWizards.ApplicationInsights.Wizard` either... do notice that there are 2 wizard extensions for `Microsoft.VisualStudio.WinRT.TemplateWizards`.

Comment (tomspilman) [inline ProjectTemplates/VisualStudio2015/WindowsUniversal/WindowsUniversal.vstemplate]: I see... we had this one extra `ApplicationInsights` wizard is all.  Makes sense now.

**Comment by BrainSlugs83** [comment]
Any chance this will be back ported to the current release? -- It's impossible to create Windows 10 UWP projects without this fix. -- It's literally just updating one zip file in the installer...

**Pull request merged**